src/platform-includes/performance/enable-automatic-instrumentation/javascript.mdx

```diff
@@ -3,7 +3,7 @@ To enable tracing, include the `BrowserTracing` integration in your SDK configur
 After configuration, you will see both `pageload` and `navigation` transactions in the Sentry UI.
 
 ```javascript
-// If you're using one of our framework SDK packages, like `@sentry/angular`,
+// If you're using one of our framework SDK packages, like `@sentry/react`,
```

Comment:
Poor Angular 😅

(no action required)

Reply:
Yeah it makes sense (also considering react vs angular usage), don't mind me rambling 😅